README.adoc

```diff
@@ -140,17 +156,22 @@ kubernetes.yaml
 include::finish/kubernetes.yaml[tags=**;]
 ----
 
-The readiness probes are configured for the containers running the `system` and `inventory` 
-microservices.
+The readiness and liveness probes are configured for the containers running the `system` 
+and `inventory` microservices.
 
-The readiness probes are configured to poll the `/health` endpoint. The readiness probe 
+The readiness probes are configured to poll the [hotspot=25 hotspot=64]`/health/ready` endpoint. The readiness probe 
 determines the READY status of the container as seen in the `kubectl get pods` output. The 
-[hotspot=27 hotspot=59]`initialDelaySeconds` field defines how long the probe should wait before it 
+[hotspot=27 hotspot=66]`initialDelaySeconds` field defines how long the probe should wait before it 
 starts to poll so the probe does not start making requests before the server has started. 
-The [hotspot=29 hotspot=61]`failureThreshold` option defines how many times the probe should fail 
-before the state should be changed from ready to not ready. The [hotspot=28 hotspot=60]`periodSeconds` 
+The [hotspot=29 hotspot=68]`failureThreshold` option defines how many times the probe should fail 
+before the state should be changed from ready to not ready. The [hotspot=28 hotspot=67]`periodSeconds` 
 option defines how often the probe should poll the given endpoint.
 
+The liveness probes are configured to poll the [hotspot=32 hotspot=71]`/health/live` endpoint. 
+The liveness probe determines when a container needs to be restarted. Similar to the readiness
```

Comment:
Change
"The liveness probe determines when a container needs to be restarted."
To
"The liveness probes determine when a container needs to be restarted."